src/tests/conftest.py

```diff
@@ -90,6 +90,7 @@
         "camera_id": 1,
         "azimuth": 43.7,
         "bucket_key": "my_file",
+        "localization": "xyxy",
```

Comment:
Mmmh, can we get a regex validation here?
Like format of each coord with X decimals, and comma-separated (or something else).

In fact, I'd like the DB initialization to fail if we put "xyxy" as a value.

Reply:
I can do it in the endpoint directly to avoid saving bad data